docs/UserGuide.adoc

```diff
@@ -161,6 +161,11 @@ Format: `clear`
 Exits the program. +
 Format: `exit`
 
+== Sorts all persons : `sort`
+
+Sorts all persons in the address book lexicographically.
```

Comment:
Good that you mentioned how the persons are sorted. 

You might want to consider putting the information for the sort command *above* the exit command. It's a very minor, or potentially not an issue at all, but it just feels weird to me. I'm typically not used to seeing anything after the *exit* command in any application (unless the exit command is directly below the command to start an application).